app/views/devise/shared/_links.html.haml

```diff
@@ -0,0 +1,19 @@
+- if controller_name != 'sessions'
+  = link_to "Sign in", new_session_path(resource_name)
```

Comment:
The links in this partial don't yet have `.govuk-link`

Reply:
good catch...